.github/workflows/measure_length_scales.yml

```diff
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  schedule:
+      - cron: "0 13 * * 1"  # Every Monday at 9AM EST
+
+jobs:
+  measure-length-scales:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup environment
+        run: |
+            python -m pip install --upgrade pip
+            pip install numpy
+            pip install imageruler
+            pip install ceviche-challenges
```

Comment:
Is this necessary?

Reply:
Yes, these are dependencies for the length scale scripts. The alternative would be to specify dependencies in a separate file (e.g. with a pyproject.toml), but since this repo is more of a collection of scripts I went for this approach.

Reply:
Why is `ceviche-challenges` a dependency?

Reply:
It’s a dependency in the script for the mode converter.

Reply:
(The mode converter script measures length scale and also evaluates the performance of the design.)